addons/website_sale/views/templates.xml

```diff
@@ -1191,7 +1190,6 @@
                 <del t-if="combination_info['compare_list_price'] and (combination_info['compare_list_price'] &gt; combination_info['price'])">
                     <bdi dir="inherit">
                     <span t-field="product.compare_list_price"
-                          groups="website_sale.group_product_price_comparison"
```

Comment:
This will only show the compare list price value anyway, so I suppose you can keep the groups no?